### ISSUE TITLE
setup a CI / CD pipeline to release the docs

### ISSUE THREAD

**Issue by jstrachan** [jenkins-x/old-documentation#1]
we've already got a static landing website: https://jenkins-x.github.io/jenkins-x-website/ hosted at github pages (we could do with a nicer CNAME mind you ;)

so it'd be good to setup a CI / CD pipeline (Jenkinsfile?) for this repo to generate the static HTML + PDF et al then copy it into the static HTML website: https://github.com/jenkins-x/jenkins-x-website then they'd appear on the landing page.

Then we'd get the static website updating on each release. 

We may then be able to get a similar CI / CD pipeline setup so that as the docs change we embed them inside the CDX UI

**Comment by rawlingsj** [comment]
Yeah - Also it would also be good to do the same for the BDD html reports that we'll be producing soon.
